src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioWorkspaceStatusServiceFactory.cs

```diff
@@ -77,6 +82,31 @@ public async System.Threading.Tasks.Task WaitUntilFullyLoadedAsync(CancellationT
 
             public Task<bool> IsFullyLoadedAsync(CancellationToken cancellationToken)
             {
+                if (_workspace.Options.GetOption(ExperimentationOptions.SolutionStatusService_ForceDelay))
```

Comment:
:exclamation: This block should not be required. Let's discuss alternatives before moving forward.

Reply:
:arrow_right: Resolved via a call with @heejaechang. I was originally interpreting "test purposes" to mean automated tests (unit / behavioral / integration), but this block was actually created with manual user experience testing in mind. The comment below has been updated to better reflect that intent.